docs/plugins/backend-plugin.md

```diff
@@ -116,36 +97,123 @@ curl localhost:7007/api/carmen/health
 
 This should return `{"status":"ok"}` like before. Success!
 
+## Secure by Default
+
+In 1.25, Backstage started moving to a secure by default model for plugins. This means that network requests to plugins will by default not allow unauthenticated users. Let's take a deeper look at the above curl request which should allow unauthenticated access.
+
+The actual endpoint that is being called is defined in
+
+```ts title="plugins/carmen-backend/src/service/router.ts"
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  // ...
+
+  // highlight-start
+  router.get('/health', (_, response) => {
+    logger.info('PONG!');
+    response.json({ status: 'ok' });
+  });
+  // highlight-end
+
+  // ...
+  return router;
+}
+```
+
+You'll notice that there is no authentication mechanism defined here, just the route name and response data. That's because the authentication is handled in your plugin definition,
+
+```ts title="plugins/carmen-backend/src/plugin.ts"
+httpRouter.use(
+  await createRouter({
+    logger,
+  }),
+);
+// highlight-start
+httpRouter.addAuthPolicy({
+  path: '/health',
+  allow: 'unauthenticated',
+});
+// highlight-end
+```
+
+This allows requests to this plugin's `/health` endpoint to go through unauthenticated!
+
+## Using Dependencies
+
+In the new backend, dependencies are defined statically during registration and then "injected" during initialization. Here's an example of what this looks like,
+
+```ts title="plugins/carmen-backend/src/plugin.ts"
+
+// highlight-start
+deps: {
+  httpRouter: coreServices.httpRouter,
+  logger: coreServices.logger,
+},
+// highlight-end
+// And then you can use them through the options property!
+// highlight-next-line
+async init({ httpRouter, logger }) {
+    // ...
+},
+
+```
+
+You can add your own dependencies by adding a named item to the `deps` parameter:
+
+```ts
+deps: {
+  // highlight-next-line
+  myDependency: coreServices.rootConfig,
+},
+```
+
+And then you can access it by referencing it in the `init` block of your plugin definition,
+
+```ts
+async init({ myDependency }) {
+   // ..
+}
+```
+
+And then you're free to call it and pass it into your router as needed.
+
+Backstage provides a bunch of `coreServices` out of box, see the more in depth docs [here](https://backstage.io/docs/backend-system/core-services/index).
```

Comment:
generally make links within the site into relative paths instead of URLs; that makes things more easily relocatable